packages/core/src/account/smartContractAccount.ts

```diff
@@ -213,7 +214,7 @@ export async function toSmartContractAccount<
 export async function toSmartContractAccount({
   transport,
   chain,
-  entryPoint,
+  entryPoint = getEntryPoint(chain),
```

Comment:
I thought we were leaving this as a required parameter based on convos in the simple account pr

Reply:
rebased to main where simple account pr is merged :)

Reply:
can we double check this? It's still an optional param here, this should be required

Reply:
doubled checked. it's not optional param, and for smartcontractaccount.ts, defaultEntrypointversion is not set as the entrypoint param is required.